twine/commands/upload.py

```diff
@@ -18,7 +18,12 @@
 from twine.package import PackageFile
 from twine import exceptions
 from twine import settings
-from twine import utils
+
+from requests.exceptions import HTTPError
+
+
+DEFAULT_REPOSITORY = "https://upload.pypi.org/legacy/"
```

Comment:
I would import these from `twine.utils` to maintain a single source, rather than redefining here. Also applies to `test_upload.py`.

Reply:
Fixed in `ea8d010`.